styles.css

```diff
@@ -538,23 +551,6 @@
     padding: 5px;
   }
 
-  #listdetails {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    list-style-type: none;
-  }
```

Comment:
- [ ] Excellent job implementing the desktop version 💯, but when you change to the mobile version, the content of the contact form is not centered, kindly, remove the default margin of a `<ul>` removing the margin left. 🐎 

![image](https://user-images.githubusercontent.com/91301423/199740466-7ab6ba77-6c56-494f-a90f-c0068a31cd4c.png)

Reply:
@fherrerao 
Please note that the reviewer before you had already approved the PR.